src/libraries/System.Linq.Expressions/tests/DebugViewTests.cs

```diff
@@ -310,13 +310,13 @@ public static void Call()
         {
             ParameterExpression x = Expression.Parameter(typeof(int), "x");
             ParameterExpression y = Expression.Parameter(typeof(int), "y");
-            ParameterExpression d = Expression.Parameter(typeof(double), "d");
+            ParameterExpression l = Expression.Parameter(typeof(long), "l");
```

Comment:
Why is this test changing?

Reply:
Because `typeof(TimeSpan).GetMethod("FromSeconds", new[] { typeof(int) })` used to pick the overload which takes `double` but now with adding the new overloads, it will pick the one that takes long.